src/tentaclio/clients/google_drive_client.py

```diff
@@ -216,5 +216,5 @@ def _build_descriptors(
                 "url": None,
             }
             if url_base is not None:
-                args["url"] = urls.URL(url_base + "/" + args["name"])
+                args["url"] = urls.URL(url_base + args["name"])
```

Comment:
you could use `os.path.join` to make it agnostic about url_base ending in "/" (as long as `args["name"]` doesn't start with /). but it looks like it's used with "/" suffixing the url_base in the code anyway so no big deal

Reply:
this wouldn't work on windows as the path separator is the infamous `\`

Reply:
ahh gotcha 👍